src/controllers/dashboard/plugins/add/index.js

```diff
@@ -22,6 +22,10 @@ function populateHistory(packageInfo, page) {
 function populateVersions(packageInfo, page, installedPlugin) {
     let html = '';
 
+    packageInfo.versions.sort((a, b) => {
+        return b.timestamp < a.timestamp ? 1 : -1;
```

Comment:
just reverse here + `[0]` instead of `.pop()` (see below)
```suggestion
        return b.timestamp < a.timestamp ? -1 : 1;
```